derive_builder/src/options/struct_mode.rs

```diff
@@ -49,6 +57,25 @@ impl OptionsBuilderMode for StructMode {
         self.builder_name(value.clone());
     }
 
+    fn parse_derive(&mut self, nested: &[syn::NestedMetaItem]) {
+        let mut traits = vec![];
+        for x in nested {
+            match *x {
+                // We don't allow name-value pairs or further nesting here, so
+                // only look for words.
+                syn::NestedMetaItem::MetaItem(syn::MetaItem::Word(ref tr)) => {
+                    traits.push(tr.clone())
```

Comment:
question is, whether we want to do any checks like `tr == "Debug"` | `"Clone"` at *this* place. But I guess it would feel a bit hacky to  have the information about `Debug` and `Clone` being automatically derived scattered about multiple places. It's probably ok, to not have any checks here.

Note: It's worth looking at how the Rust compiler complains about this

# IN
```rust
#[derive(Builder)]
#[builder(derive(Default))]
struct Lorem {
```

# OUT
```
error[E0119]: conflicting implementations of trait `std::default::Default` for type `LoremBuilder`:
 --> derive_builder/tests/derive_trait.rs:7:10
  |
7 | #[derive(Builder)]
  |          -------
  |          |
  |          conflicting implementation for `LoremBuilder`
  |          first implementation here
```

So the error message is ok-ish, but not perfect. I think it would be nicer, if this would either just work without error, or if the builder would panic with a custom message.